### ISSUE TITLE
Support TypeScript v4

### ISSUE THREAD

**Issue by davidhouweling** [toshi-toma/eslint-config-airbnb-typescript-prettier#64]
At present the peer dependencies say `<3.10.0`, any reason the peer dependencies couldn't simply be `>=3.3.1` without an upper bound?

**Comment by dohomi** [comment]
CRA is nearly unusable with this configuration set. @toshi-toma are you planning to update the dependencies? I think especially 
```
"@typescript-eslint/eslint-plugin": "^3.3.0",
"@typescript-eslint/parser": "^3.3.0",
```
causing error with no-unused vars. They would be need to be upgraded to v4

**Comment by toshi-toma** [comment]
@davidhouweling @dohomi 

`eslint-config-airbnb-typescript-prettier` has just supported TypeScript v4. #67 

It is released as v4.0.0

Thank you @dohomi
